src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/writer/EventLogQueueProcessor.java

```diff
@@ -114,9 +116,8 @@ public void purgeQueueAndPersist() {
             } else if (entry.epoch == nextTimeBucket) {
                 nextMetrics.add(entry);
             } else {
-                LOG.info("UNEXPECTED entry ({}) with epoch '{}' arrived" +
```

Comment:
nit: let's add a few comments here to explain why metrics can fall behind the current bucket and what PMET is added to monitor the issue

Reply:
I've added a comment as suggested, thanks.